<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab2_CropStatus</class>
 <widget class="QWidget" name="Tab2_CropStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,4,2,0">
     <item>
      <widget class="QDateTimeEdit" name="pStartdatetime_crop"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="pEnddatetime_crop"/>
     </item>
     <item>
      <widget class="QPushButton" name="pPBcrop">
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBclearcrop">
       <property name="text">
        <string>초기화</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="pTWcrop">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fresh</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unripe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rotten</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
